### PR TITLE
Python: Add `PFADD` command.

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
ref https://github.com/aws/glide-for-redis/pull/1221

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Node: Added ZRANGE command ([#1115](https://github.com/aws/glide-for-redis/pull/1115))
 * Python: Added RENAME command ([#1252](https://github.com/aws/glide-for-redis/pull/1252))
 * Python: Added APPEND command ([#1152](https://github.com/aws/glide-for-redis/pull/1152))
+* Python: Add `PFADD` command (TODO PR #)
```
Comment (aaron-congo) [inline CHANGELOG.md:11]: not a big deal but I'd say lets follow the existing format
```suggestion
* Python: Added PFADD command (TODO PR #)
```

File: python/python/glide/async_commands/core.py [hunk 2]
```diff
@@ -2041,3 +2041,32 @@ async def invoke_script(
...
+        Creates a new structure if the `key` does not exist.
+
+        When no `elements` are provided, and `key` exists and is a HyperLogLog, then no operation is performed.
+        If `key` does not exist, then the HyperLogLog structure is created.
+
+        See https://redis.io/commands/pfadd/ for more details.
+
+        Args:
+            key (str): The `key` of the HyperLogLog data structure to add elements into.
+            elements (List[str]): An array of members to add to the HyperLogLog stored at `key`.
```
Comment (aaron-congo) [inline python/python/glide/async_commands/core.py:2057]:
```suggestion
            elements (List[str]): An list of members to add to the HyperLogLog stored at `key`.
```

File: python/python/glide/async_commands/transaction.py [hunk 3]
```diff
@@ -1551,6 +1551,26 @@ def dbsize(self: TTransaction) -> TTransaction:
...
+        Creates a new structure if the `key` does not exist.
+
+        When no `elements` are provided, and `key` exists and is a HyperLogLog, then no operation is performed.
+        If `key` does not exist, then the HyperLogLog structure is created.
+
+        See https://redis.io/commands/pfadd/ for more details.
+
+        Args:
+            key (str): The `key` of the HyperLogLog data structure to add elements into.
+            elements (List[str]): An array of members to add to the HyperLogLog stored at `key`.
```
Comment (aaron-congo) [inline python/python/glide/async_commands/transaction.py:1566]:
```suggestion
            elements (List[str]): An list of members to add to the HyperLogLog stored at `key`.
```

File: python/python/tests/test_transaction.py [hunk 4]
```diff
@@ -32,6 +32,7 @@ async def transaction_test(
     key7 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key8 = "{{{}}}:{}".format(keyslot, get_random_string(3))
     key9 = "{{{}}}:{}".format(keyslot, get_random_string(3))
+    key11 = "{{{}}}:{}".format(keyslot, get_random_string(3))  # hyper hyper log log
```
Comment (aaron-congo) [inline python/python/tests/test_transaction.py:35]:
```suggestion
    key11 = "{{{}}}:{}".format(keyslot, get_random_string(3))  # hyper log log
```

File: CHANGELOG.md (hunk 1 above)
Comment (Yury-Fridlyand) [reply CHANGELOG.md:11]: Should be there a feature name or PR title?

Comment (aaron-congo) [reply CHANGELOG.md:11]: Took a look at their PR titles and they don't necessarily match the CHANGELOG entry exactly. I'd match the existing format here though, and then the PR title can either be the same or slightly different.

**Review by aaron-congo — APPROVED** [review]

**Pull request merged**